workspace/__lib__/xod/core/throttle/patch.cpp

```diff
@@ -12,10 +12,13 @@ struct State {
 {{ GENERATED_CODE }}
 
 void evaluate(Context ctx) {
-    if (isSettingUp()) return;
+    auto inValue = getValue<input_IN>(ctx);
+    if (isSettingUp() && isInputDirty<input_IN>(ctx)) {
```

Comment:
I'm not sure there should be a special case for `isSettingUp` at all.

Reply:
So, what should be the starting output value? Something bound to output (which we are going to make immutable someday) or the initial value of the input? I think the latter has more sense. Anyway, I can’t understand why the `isInputDirty<input_IN>(ctx)` check is necessary in addition to `isSettingUp`?